File: server/herehear/src/main/java/com/ssafy/herehear/db/entity/Book.java
package com.ssafy.herehear.db.entity;

import javax.persistence.Entity;

import org.hibernate.annotations.ColumnDefault;

import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
public class Book extends BaseEntity {
	private String isbn13;	
	private String title;	
	private String author;	
	private String description;	
	private String publisher;	
	private String img_url;	
	
	@ColumnDefault("0")
	private int stars_sum;
	
	@ColumnDefault("0")
	private int stars_count;

}

File: client/herehear/app/src/main/java/com/ssafy/herehear/data/network/response/Book.kt
package com.ssafy.herehear.data.network.response

data class Book(
    val author: String,
    val description: String,
    val id: Int,
    val img_url: String,
    val isbn13: String,
    val publisher: String,
    val stars_count: Int,
    val stars_sum: Int,
    val title: String
)